### PR TITLE
UI/15528255 profile page settings

### PR DESCRIPTION
**Pull request by krawczykkatarzyna**
Screen Shot 2023-07-18 at 15 31 21

### CODE REVIEW

File: src/components/v5/common/Fields/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  return (
+    <div className="flex flex-col gap-1">
+      <div className="flex relative">
+        <input
+          type="text"
+          {...register?.(name)}
+          name={name}
+          placeholder={placeholder}
+          className={clsx('input-round input', {
```
Comment (CzarekDryl) [inline src/components/v5/common/Fields/Input/Input.tsx:40]: What are those classes?

File: .storybook/main.css [hunk 2]
```diff
@@ -93,6 +93,18 @@
   .navigation-link {
     @apply text-gray-700 w-full flex items-center transition-all duration-normal py-2 heading-5 hover:text-blue-400 sm:font-normal sm:text-md sm:text-gray-900;
   }
+
+  .subnav-button {
+    @apply flex w-full items-center text-md hover:bg-gray-50 hover:font-medium rounded py-2 px-[1.2rem];
```
Comment (KatarzynaStefanowiczPagepro) [inline .storybook/main.css:98]: px-[1.2rem] isn't a full value, please correct it

File: src/components/frame/v5/pages/UserProfilePage/partials/LeftColumn.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { LeftColumnProps } from '../types';
+
+const displayName = 'v5.pages.UserProfilePage.partials.LeftColumn';
+
+const LeftColumn: FC<LeftColumnProps> = ({ fieldTitle, fieldDecription }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <div className="flex flex-col w-[16.375rem]">
+      <span className="text-gray-900 text-1 leading-5">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/v5/pages/UserProfilePage/partials/LeftColumn.tsx:13]: Do you need this style text-gray-900?

Comment (KatarzynaStefanowiczPagepro) [reply src/components/frame/v5/pages/UserProfilePage/partials/LeftColumn.tsx:13]: Maybe it should be w-full max-w-[16.375rem] on mobile devices?

File: src/components/v5/common/Fields/Input/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import FormError from '~v5/shared/FormError';
+import { InputProps } from './types';
+import { useInput } from '../hooks';
+import { DEFAULT_MAX_CHAR_NUMBER } from './consts';
+
+const displayName = 'v5.common.Textarea';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Input/Input.tsx:10]: Please, correct displayName

File: src/components/v5/common/Fields/Input/Input.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    <div className="flex flex-col gap-1">
+      <div className="flex relative">
+        <input
+          type="text"
+          {...register?.(name)}
+          name={name}
+          placeholder={placeholder}
+          className={clsx('input-round input', {
+            'border-gray-300': !isTyping,
+            'border-blue-200 shadow-lightBlue': isTyping,
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Input/Input.tsx:42]: Shouldn't be shadow-light-blue and added in the theme?

File: .storybook/main.css (hunk 2 above)
Comment (KatarzynaStefanowiczPagepro) [reply .storybook/main.css:98]: Are these styles added to main.global.css?

File: src/components/frame/v5/pages/UserProfilePage/hooks.tsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+          type="success"
+          title={{ id: 'user.profile.toast.title.success' }}
+          description={{
+            id: 'user.profile.toast.description.success',
+          }}
+        />,
+      );
+    } catch (err) {
+      toast.error(
+        <Toast type="error" title="Error" description="Something went wrong" />,
```
Comment (CzarekDryl) [inline src/components/frame/v5/pages/UserProfilePage/hooks.tsx:78]: Please add translation

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

**Review by CzarekDryl — APPROVED** [review]

**Pull request merged**